rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1017,6 +1017,49 @@ private void logUpdateCollections(Transaction rskTx) {
         eventLogger.logUpdateCollections(sender);
     }
 
+    protected void processValidationFailure(Federation proposedFederation) {
```

Comment:
```suggestion
    protected void processSVPFailure(Federation proposedFederation) {
```
What do you think? To give it context since it's part of BridgeSupport class and there's all of type of stuff in there

Reply:
makes sense, thanks!

Reply:
updated